docs/getting-started.md

```diff
@@ -177,60 +190,73 @@ Claim {
 ```
 
 The `requestForAttestation` object looks like this:
 
 ```typescript
 RequestForAttestation {
-  claim:
-   Claim {
-     cType:
-      '0x5a9d939af9fb5423e3e283f16996438da635de8dc152b13d3a67f01e3d6b0fc0',
-     contents: { name: 'Alice', age: 29 },
-     owner: '5EvSHoZF23mZS4XKQBLdqMv7a7CRSANJmxn7XDu6hwoiK4Wz' },
-  ctypeHash:
-   { nonce: '1f81c5b4-6765-461f-8748-cc6682ea0dcb',
-     hash:
-      '0xf209e07c39502db88dbdcbe409b5c13f70204afab7a3dd8fbeafa0e4f46cf694' },
+  claim: Claim {
+    cTypeHash: '0xd8ad043d91d8fdbc382ee0ce33dc96af4ee62ab2d20f7980c49d3e577d80e5f5',
+    contents: { name: 'Alice', age: 29 },
+    owner: '5HTEzvVT5bQxJTYPiDhRUw4GHarQVs66sFQEpQDUNT6MyoJr'
+  },
+  claimOwner: {
+    nonce: '4ef65c7b-ee10-4068-a53e-f6a1bf8bc5f1',
+    hash: '0x251394fa14525606fdfee2d0a352589a413e74256d13472284be01003dcf0b4a'
+  },
+  cTypeHash: {
+    nonce: '26bc33bf-52ef-422f-b321-b82e11e0b207',
+    hash: '0xde3adcce5ce51b1f1ffbbede9e35facedd2de4b2781d86164bea212bbc13eaa4'
+  },
   legitimations: [],
-  delegationId: undefined,
-  claimHashTree:
-   { name:
-      { nonce: '1b0cada9-7d0b-4ef5-b55f-2cd0b81da1fa',
-        hash:
-         '0x68e8292dd15f3965a84c5e7be317abbba9aef5be242decfd763f357b205a755a' },
-     age:
-      { nonce: 'f8b252e9-ebc5-4f1b-bda5-4b282311dfba',
-        hash:
-         '0xb135bf42e7d324b572a0da15a6d7c6302a086493f4c9eee661ca8b8266eda8fd' } },
-  hash:
-   '0x2b3f7c8b44fb42d0cab1cab63f22ed92841db4ab169f7c7c60aa1ece10eaf5b4',
-  claimerSignature:
-   '0x3de3b6c245f43533a9f78730dc9f32664098adec5e31ae643f826b2439c00fa18720e1c40dcfde3a99eda74903e5be09303096286ef7659ba312a3b4a807550b' }
+  delegationId: null,
+  claimHashTree: {
+    name: {
+      nonce: 'a3c8b829-1a55-4e65-b09f-3a7f0b8be1ff',
+      hash: '0xf0cd53c91e36d2747d7b911b3a90992dc591315126e076c05cb93ab72ee9f893'
+    },
+    age: {
+      nonce: 'ed6fd1e2-d292-4dbe-b5d4-1147abf73ca4',
+      hash: '0x1f029ea4cb82e24e9b1ece1481b526b37a198e2f9279759e17b5b1c2a5c832af'
+    }
+  },
+  rootHash: '0xb672fe6fb46985459ee5efe67f6610056a85b3b4283cc4ead6bf07e9fd1c27e6',
+  claimerSignature: '0x0043f4a404ea3930cffa4d74d00aacddf8f4f10c1af281c792d9f0e49153a40cf96f35f18ee2676279a2c96173ee183a2a032c2e30feac1ac9fbc64c535491e306',
+  privacyEnhancement: null
+}
 ```
 
-To send the request for an attestation to an Attester, first we need to create an Attester identity:
+#### 5.1.2. Sending an Attestation
 
-```typescript
-import Kilt from '@kiltprotocol/sdk-js'
+Before we can send the request for an attestation to an Attester, we should first [create an Attester identity like above](#how-to-generate-an-identity).
 
-const mnemonicForAttester = Kilt.Identity.generateMnemonic()
-const attester = Kilt.Identity.buildFromMnemonic(mnemonicForAttester)
+```typescript
+const attesterMnemonic = Kilt.Identity.generateMnemonic()
+// mnemonic: coast ugly state lunch repeat step armed goose together pottery bind mention
```

Comment:
this step is not necessary. If no mnemonic is provided, a new mnemonic is generated anyways. 

But then you don't have access to the mnemonic. OK maybe leave it like that... I think it's useless to create a mnemonic inside `buildFromMnemonic`. Most of the time you would need to store the mnemonic and not throw it away... Maybe this should be put into an issue.

Reply:
> But then you don't have access to the mnemonic. OK maybe leave it like that... I think it's useless to create a mnemonic inside `buildFromMnemonic`. Most of the time you would need to store the mnemonic and not throw it away... Maybe this should be put into an issue.

It's used the same way in the tutorial and I agree that in the future we should remove the auto mnemonic generation when not supplying one. It improves devX but can lead to problems for users.